src/main/scala/com/srcd/swivel/SparkPrepDriver.scala

```diff
@@ -0,0 +1,415 @@
+package com.srcd.swivel
```

Comment:
Thank for suggestion, sounds good, will do that as a last step, after all other reviews are addressed.